partner-portal/src/API/ViewModels/CaseCallback.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace Rsbc.Dmf.PartnerPortal.Api.ViewModels
 {
-    public class Callback
+    public class CaseCallback
```

Comment:
I would rename this back to Callback. Makes more sense to me and would be more consistent

Reply:
This is causing namespace issues on the controller level and CMS service level. Need to fix the namespaces on this but renaming to caseCallback also makes sense as this happens at case level. WE have this error in driver portal as well

Reply:
add a TODO comment here to change this back to "Callback" after we change both driver portal and partner portal DriverController.Create. The Create controller service should be using a new model e.g. CallbackRequest. The only time proto models should be used are returning objects back from gRPC clients

Reply:
I added a TODO comment for now. Still the API needs to be integrated for the callbacks  I will fix his as as a part of callback feature.